stores.js

```diff
@@ -0,0 +1,179 @@
+'use strict'
+const {
+  crypto_generichash: hash,
+  crypto_sign_keypair: createKeypair,
+  crypto_sign_verify_detached: verify,
+  crypto_sign_detached: sign,
+  crypto_sign_PUBLICKEYBYTES: pkSize,
+  crypto_sign_SECRETKEYBYTES: skSize,
+  crypto_sign_BYTES: signSize
+} = require('sodium-universal')
+const { finished } = require('readable-stream')
+const { MutableStore } = require('./messages')
+
+const mutableEncoding = {
+  encode (o) {
+    if (o == null) return o
+    return MutableStore.encode(o)
+  },
+  decode (o) {
+    if (o == null) return o
+    return MutableStore.decode(o)
+  }
+}
+
+const PUT_VALUE_MAX_SIZE = 1000
```

Comment:
Add a comment that PUT_VALUE_MAX_SIZE + packet overhead (ie the key etc) should be < the network MTU which is normally 1400 bytes